### ISSUE TITLE
isInvalidAccessibility.js throws  `TypeError: Cannot convert undefined or null to object`

### ISSUE THREAD

**Issue by L0wry** [applitools/eyes-storybook#28]
Currenly unable to run storybook eyes due to 
TypeError: Cannot convert undefined or null to object
 version : "@applitools/eyes-storybook": "2.8.6",

this thanks to `Object.values` on `AccessibilityRegionType` which is null
```
use strict';
const { TypeUtils } = require('@applitools/eyes-common');
const { AccessibilityRegionType } = require('@applitools/eyes-common');

function isInvalidAccessibility(accessibility = []) {
    console.log({ type: AccessibilityRegionType });
    const accObjects = [].concat(accessibility);
    const err = [];
    const typeMsg = `Valid accessibilityType values are: ${Object.values(AccessibilityRegionType)}`;
    for (const acc of accObjects) {
        if (
            acc.accessibilityType &&
            !TypeUtils.has(AccessibilityRegionType, acc.accessibilityType)
        ) {
            err.push(
                `The region ${JSON.stringify(acc)} has an invalid accessibilityType of: ${
                    acc.accessibilityType
                } `
            );
            !err.length && err.unshift(typeMsg);
        }
    }
    return err.join('\n');
}

module.exports = isInvalidAccessibility;

```

results in the following log 

```
✔ Storybook was started
✔ Reading stories
⠴ Done 0 stories out of 1{ type: undefined }
✖ Done 1 stories out of 1
undefined [ TypeError: Cannot convert undefined or null to object
      at Function.values (<anonymous>)
      at isInvalidAccessibility (/Users/matthewlowry/dev/bp/smp-carama-workshop-ui/node_modules/@applitools/eyes-storybook/node_modules/@applitools/visual-grid-client/src/sdk/isInvalidAccessibility.js:9:67)
      at checkWindow (/Users/matthewlowry/dev/bp/smp-carama-workshop-ui/node_modules/@applitools/eyes-storybook/node_modules/@applitools/visual-grid-client/src/sdk/checkWindow.js:69:24)
      at timeItAsync (/Users/matthewlowry/dev/bp/smp-carama-workshop-ui/node_modules/@applitools/eyes-storybook/src/renderStory.js:39:13)
      at process._tickCallback (internal/process/next_tick.js:68:7) ]
```

**Comment by danielschwartz85** [comment]
Hi @L0wry,
Can u share your ```@applitools/visual-grid-client``` and ```@applitools/eyes-common``` versions ?
```
 npm ls | grep  @applitools
```
Tnx.

**Comment by amitzur** [comment]
No activity - closing issue.
This repository is going to be archived and moved into the monorepo of all Applitools SDK's.
For new issues on this SDK, please file them at https://github.com/applitools/eyes.sdk.javascript1
